cli/README.md

```diff
@@ -61,6 +61,16 @@ $ kubectl kadalu storage-add storage-pool-3 \
     --pvc azure-disk-1
 ```
 
+To `archive` persistent volume claims,
+> Note:
+> PV Reclaim Policy is set to `delete` by default
+
+```
+$ kubectl kadalu storage-add storage-pool-1 \
+    --pvReclaimPolicy=archive
```

Comment:
For uniformity, please change to `--pv-reclaim-policy`

Reply:
sure

Reply:
--pv-reclaim-policy
```
vatsa@vatsa-VirtualBox:~/kadalu/cli$ sudo build/kubectl-kadalu storage-add --device=vatsa-virtualbox:/mnt/disk1 storage-pool-1 --pv-reclaim-policy=archive
The following nodes are available:
  vatsa-virtualbox

Storage Yaml file for your reference:

apiVersion: "kadalu-operator.storage/v1alpha1"
kind: "KadaluStorage"
metadata:
  name: "storage-pool-1"
spec:
  type: "Replica1"
  storage:
    - node: "vatsa-virtualbox"
      device: "/mnt/disk1"
  pvReclaimPolicy: archive

Is this correct?(Yes/No): 
```